index.md

```diff
@@ -67,6 +67,7 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 2 | howto/manage/web-dashboard | [Use the web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871)|
 | 2 | howto/application/landing | [Manage applications](https://discourse.ubuntu.com/t/manage-applications/24333) |
 | 3 | howto/application/create | [Create an application](https://discourse.ubuntu.com/t/create-an-application/24198)|
+| 3 | howto/application/stream | [Stream an application](tbd)|
```

Comment:
Would this be updated after you post the stream.md on discourse ?

Reply:
yes, for new topics, we get the link after we post to discourse and then update it.